media-libs/fblend/fblend-0.4.recipe

```diff
@@ -1,31 +1,53 @@
+SUMMARY="Color blenders for Allegro"
 DESCRIPTION="FBlend is a series of super-fast color blenders for Allegro."
 HOMEPAGE="http://sourceforge.net/projects/fblend/"
-SOURCE_URI="http://sourceforge.net/projects/fblend/files/FBlend%20beta/fblend-0.5-beta01/fblend-0.5-beta01.zip/download"
-REVISION="1"
-STATUS_HAIKU="broken"
-DEPEND="media-libs/allegro >= 4.4
-	app-text/dos2unix >= 1.0"
-CHECKSUM_MD5="e252ae521e099793fe6caa0fb7d048f3"
+COPYRIGHT="2001-2002 Robert J Ohannessian"
+LICENSE="BSD (3-clause)"
+REVISION="2"
+SOURCE_URI="https://sf.net/projects/fblend/files/FBlend/0.4/fblend-0.4.zip/download"
```

Comment:
You can use the redirector service:
```
SOURCE_URI="https://downloads.sf.net/fblend/fblend-$portVersion.zip"
```
Reminder: SourceForge redirects all https downloads to http, so using https is not safer than http. But maybe SF will start serving https for their downloads too, some day.

BTW, you can switch HOMEPAGE to https (because SF does serve sourceforge.net/projects/... through https).